src/main/scala/scorex/core/network/peer/PeerManager.scala

```diff
@@ -23,7 +23,11 @@ class PeerManager(settings: ScorexSettings, scorexContext: ScorexContext) extend
 
   if (peerDatabase.isEmpty) {
     // fill database with peers from config file if empty
-    settings.network.knownPeers.foreach { address =>
+    fillPeerDatabase(settings.network.knownPeers)
+  }
```

Comment:
we should also fire a GetNewPeers here , to first start the process at bootstrap